build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     ext {
-        gradle_plugins_version = '4.0.38-SNAPSHOT'
+        gradle_plugins_version = '4.0.40-SNAPSHOT'
```

Comment:
I think this change must have been accidental. We only change the version number at release time.

Reply:
done done.